torch/csrc/jit/python/pybind_utils.h

```diff
@@ -46,6 +46,26 @@
 namespace torch {
 namespace jit {
 
+// The PythonFutureWrapper for ivalue::Future
+struct PythonFutureWrapper {
+  explicit PythonFutureWrapper(c10::intrusive_ptr<c10::ivalue::Future> fut)
+      : fut(std::move(fut)) {}
+
+  IValue wait() {
+    fut->wait();
```

Comment:
This line is new? Will this change trace behavior?

Reply:
no it will not, tested locally it works fine. Before this PR tracer directly return `fut->value()`, `wait` is a no-op in tracing mode.